fe/fe-core/src/main/java/org/apache/doris/nereids/rules/exploration/mv/AbstractMaterializedViewAggregateRule.java

```diff
@@ -53,6 +64,17 @@
  */
 public abstract class AbstractMaterializedViewAggregateRule extends AbstractMaterializedViewRule {
 
+    protected static final Map<PlaceholderExpression, PlaceholderExpression>
+            AGGREGATE_ROLL_UP_EQUIVALENT_FUNCTION_MAP = new HashMap<>();
+    protected final String currentClassName = this.getClass().getSimpleName();
+    private final Logger logger = LogManager.getLogger(this.getClass());
+
+    static {
+        AGGREGATE_ROLL_UP_EQUIVALENT_FUNCTION_MAP.put(
+                PlaceholderExpression.of(Count.class, 0, true),
+                new PlaceholderExpression(ImmutableList.of(), BitmapUnion.class, 0));
```

Comment:
do not use PlaceholderExpression as we negotiate offline

Reply:
have fixed it